src/definitions/modules/calendar.js

```diff
@@ -419,11 +419,11 @@
                                     cell = $('<td/>').addClass(className.cell).appendTo(row);
                                     cell.text(cellText);
                                     cell.data(metadata.date, cellDate);
-                                    var adjacent = isDay && cellDate.getMonth() !== ((month + 12) % 12);
-                                    var disabled = (!settings.selectAdjacentDays && adjacent) || !module.helper.isDateInRange(cellDate, mode) || settings.isDisabled(cellDate, mode) || module.helper.isDisabled(cellDate, mode) || !module.helper.isEnabled(cellDate, mode);
-                                    var eventDate;
+                                    let adjacent = isDay && cellDate.getMonth() !== ((month + 12) % 12);
+                                    let disabled = (!settings.selectAdjacentDays && adjacent) || !module.helper.isDateInRange(cellDate, mode) || settings.isDisabled(cellDate, mode) || module.helper.isDisabled(cellDate, mode) || !module.helper.isEnabled(cellDate, mode);
+                                    let eventDate = null;
```

Comment:
I could not get eslint to autofix the original `var` here, any idea why?

(all other non-auto fixed places required legit manual work or were related with https://github.com/eslint/eslint/issues/16610)

Reply:
Perhaps eslint wants `let` to always have an assignment?